scripts/v2-migration/main.go

```diff
@@ -0,0 +1,300 @@
+package main
+
+import (
+	"bytes"
+	_ "embed"
+	"flag"
+	"fmt"
+	"os"
+	"path"
+	"sort"
+	"strings"
+	"text/template"
+	"time"
+
+	"github.com/cloudquery/cloudquery/scripts/v2-migration/internal/convert"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/gertd/go-pluralize"
+)
+
+var (
```

Comment:
This is a once-off script that we'll throw away in a week or two, so I didn't spend too much time cleaning it up. Please forgive stylistic issues :bow: